library/src/test/java/bio/terra/landingzone/library/landingzones/management/ResourcesReaderImplTest.java

```diff
@@ -1,206 +0,0 @@
-package bio.terra.landingzone.library.landingzones.management;
```

Comment:
This test use obsolete code to create LZ. Also we have some list resource validation in our integration test.